### DISCUSSION TITLE
update collection without updating vectors.

### DISCUSSION THREAD

**Discussion by amzker** [milvus-io/milvus#36558] [Q&A and General discussion] [answered]
so i need to update the records in collection but not the vectors , do i need to remake all the vectors?

**Comment by yhmo** [comment, marked as answer]
In current versions, yes, to update one row, you need to update the entire row, including the vector field.
In our plan, partial update ability will be available in Milvus v3.0.

**Reply by amzker** [reply]
Thanks

**Reply by xiaofan-luan** [reply]
This is something we want to cover in 3.0.
Mark it

**Comment by amzker** [comment]
any updates on this? or as we are still on 2.5.3 there is no changes in this

**Reply by yhmo** [reply]
Partial update is still in progress, in our plan, this feature will be available in a minor version of v2.5.x

**Reply by amzker** [reply]
Thanks for the update
